tools/include/context.hpp

```diff
@@ -20,19 +20,18 @@ inline namespace MIGRAPHX_INLINE_NS {
 struct context
 {
     /// Wait for any tasks in the context to complete
-    void finish() const;
+    void finish();
 };
 
 #else
 
 <%
 interface('context',
-    virtual('finish', returns='void', const=True)
```

Comment:
Why does the `const` need to be removed?

Reply:
will add "const" back after current rounds of Jenkins passed,   This is likely relics of earlier version when event is destroyed in finish.

Reply:
fixed.